### PR TITLE
Log every non root new process started/failed on the endpoint

### PR DESCRIPTION
**Pull request by chajain**
Log every new process started/failed on the endpoint

### CODE REVIEW

**Review by guangyee — APPROVED** [review]
Looks good! Thanks Charu!

File: artifacts/definitions/Linux/Events/ProcessStatuses.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+       WHERE "vrr_proexec" in Tags
+
+     // Cache Uid -> Username mapping.
+     LET usrs <= SELECT User, atoi(string=Uid) AS Uid
+       FROM Artifact.Linux.Sys.Users()
+
+     // Enrich the original artifact with more data.
+     SELECT Time, Pid, Ppid, UserId,
+              { SELECT User from usrs WHERE Uid = UserId} AS User,
+              regex_replace(source=read_file(filename= "/proc/" + PPID + "/cmdline"),
```
Comment (Firstyear) [inline artifacts/definitions/Linux/Events/ProcessStatuses.yaml]: What happens if the process is dead before we got to read this from proc?

**Review by Firstyear — CHANGES_REQUESTED** [review]

**Review by Firstyear — APPROVED** [review]

File: artifacts/definitions/Linux/Events/ProcessStatuses.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    description: We depend on augenrules to load the process execution rules.
+
+sources:
+  - query: |
+     // Install the auditd rule if possible.
+     // Remove default filter if any
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_clear_default.rules ]; then echo '-d task,never -i' > /etc/audit/rules.d/vrr_clear_default.rules; chmod 0640 /etc/audit/rules.d/vrr_clear_default.rules; " + pathToAugenrules + " --load; fi"])
+
+     // add rules to collect new process 
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_proexec.rules ]; then echo -e '-a always,exit -F arch=b64 -S execve -F euid!=0 -F auid!=-1 -k vrr_proexec\n-a always,exit -F arch=b32 -S execve -F euid!=0 -F auid!=-1 -k vrr_proexec' >> /etc/audit/rules.d/vrr_proexec.rules; chmod 0640 /etc/audit/rules.d/vrr_proexec.rules; " + pathToAugenrules + " --load; fi"])
```
Comment (guangyee) [inline artifacts/definitions/Linux/Events/ProcessStatuses.yaml]: `>` instead of `>>` as you are only writing the file once

**Comment by chajain** [comment]
https://github.com/SUSE/linux-security-sensor/pull/34

**Pull request closed without merging**